docs/manage-your-account/creating-environments.mdx

```diff
@@ -0,0 +1,240 @@
+---
+
+import createEnvironment from "/ui-videos/workspace-settings/creating-new-environment.mp4";
+
+
+# Create a new Environment
+
+Creating a new environment is as simple as clicking the **New Environment** button and filling in the details.
+
+<video controls>
+  <source src={createEnvironment} />
+</video>
+
+## Creating Environment via API
+
+You can also create a new environment via the API.
+
+```bash
+curl 'https://api.permit.io/v2/projects/<project-id>/envs' \
+  -H 'authorization: Bearer <api-key>' \
+  --data-raw '{"key":"new-env-name","name":"New Env Name"}'
+```
+
+:::info Remember
+Make sure to use an Organization-level API Key or Project-level API Key to create a new Environment.
+Read more about [API Key levels](/manage-your-account/workspace-settings#api-keys).
+:::
+
+
+#### New Environment's Default Settings
+
+When creating a new environment, the following resources and roles will be created by default:
+```json
+{
+  "default_resource_actions": ["create", "read", "update", "delete"],
+  "default_resource_permissions": {
+    "admin": ["create", "read", "update", "delete"],
+    "editor": ["create", "read", "update"],
+    "viewer": ["read"]
+  },
+  "enable_default_roles": true
+}
+```
+
+You can disable the default roles by setting `enable_default_roles` to `false` in the request body.
+```bash
+curl 'https://api.permit.io/v2/projects/<project-id>/envs' \
+  -H 'authorization: Bearer <api-key>' \
+  --data-raw '{"key":"new-env-name","name":"New Env Name", "settings": {"enable_default_roles": false}}'
+```
+
+
+## Coping / Merging Environments (EAP)
+
+Environments can be copied into new environments or existing environments.
+This is useful when you want to manage changes to the **Policy**, like adding a Resource, Role or Condition Set, in a controlled manner.
+
+For example, if you want to test adding a new Resource to your policy, you can copy the environment, make the changes, and test them on the new environment, then merge it back.
+
+```mermaid
+%%{init: { 'gitGraph': {'mainBranchName': 'Production'}} }%%
+gitGraph
+    commit id: "Add File Resource"
+    commit id: "Add Editor Role"
+    branch Development
+    commit id: "Add Folder Resource"
+    commit id: "Add Admin Role"
+    checkout Production
+    merge Development
+    commit id: "Add Viewer Resource"
+```
+
+In this example, we have a Production environment with a policy that has a `File` Resource and an `Editor` Role.
+We've copied the Production environment into a Development environment, where we've added a `Folder` Resource and an `Admin` Role.
+After testing the changes, we've merged the Development environment back into the Production environment.
+
+This way, you can manage changes to your policy like code changes in your CI/CD pipelines.
+Read more about [Policy Life Cycle with Permit](/how-to/SDLC/CI-CD).
+
+:::warning Notice
+This feature is in **EAP** stage, which means it's in early access and might have some limitations.
```

Comment:
No need to push this EAP to every place with long warnings
Put one info on this and in the rest of the places EAP is enough